File: C++/Estrutura condicional/Troco_verificado.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    double preco, dinheiro, troco, valorTot, falta;
    int qtde;

    cout << "Preco unitario do produto:";
    cin >> preco;
    cout << "Quantidade comprada: ";
    cin >> qtde;
    cout << "Dinhheiro recebido: ";
    cin >> dinheiro;

    valorTot = preco * qtde;

    cout << fixed << setprecision(2);
    if (dinheiro < valorTot)
    {
        falta = valorTot - dinheiro;

        cout << "DINHEIRO INSUFICIENTE, FALTAM " << falta << " REAIS" << endl;
    }
    else
    {
        troco = dinheiro - valorTot;
        cout << "TROCO = " << troco << endl;
    }

    return 0;
}

File: Python/Estrutura Condicional/Troco_verificado.py
preco: float; dinheiro: float; troco: float; valor: float
qtde: int

preco = float(input("Preco unitario do produto: "))
qtde = int(input("Quantidade comprada: "))
dinheiro = float (input("Dinheiro recebido: "))

valor = preco * qtde

if dinheiro < valor:
    falta = valor - dinheiro
    print(f"DINHEIRO INSUFICIENTE, FALTAM {falta:.2f} REAIS")
else:
    troco = dinheiro - valor
    print(f"TROCO = {troco:.2f}")
